### PR TITLE
Allow condenser to run against different chains

### PR DESCRIPTION
**Pull request by jnordberg**
This patch adds config options for setting a custom chain id and address prefix. I also had to update the browser tests for it to run.

### CODE REVIEW

File: app/Main.js [hunk 1]
```diff
@@ -26,6 +26,12 @@ function runApp(initial_state) {
     console.log('Initial state', initial_state);
     const config = initial_state.offchain.config
     steem.config.set('websocket', config.ws_connection_client);
+    if (config.custom_addr_prefix) {
+        steem.config.set('address_prefix', config.custom_addr_prefix);
```
Comment (sneak) [inline app/Main.js]: i would recommend setting address_prefix to `STM` in defaults.json, and dropping this logic and the use of the `custom_` prefixes.  Then to use a 'custom' one you just change address_prefix.

**Comment by jnordberg** [comment]
Done

**Comment by roadscape** [comment]
Why remove the memo encryption test?

**Comment by jnordberg** [comment]
@roadscape When the memo test runs a very similar test was run internally in steem-js, with a hardcoded `STM` prefix causing the test case to fail.

As I understood the code, the intention of the test was just to check availability of memo encryption, not throw and stop people from logging in. But since the internal steem-js test runs and throws failing the test case it defeated that purpose, so I figured it was better to remove since it never worked in the first place

**Comment by roadscape** [comment]
@valzav is it safe to remove this test? I'm not sure of the status of memo encryption myself.

**Comment by valzav** [comment]
@roadscape, I think it's safe to remove memo encryption test - memo encryption is not used in condenser anyway

**Pull request merged**